assets/json/pattern.json

```diff
@@ -7,7 +7,7 @@
   ".*materialize.*\\.js$": "",
   ".*mootools.*\\.js$": "",
   ".*require.*\\.js$": "",
-  ".*vimrc.*": "",
+  ".*vimrc[^/\\\\]*$": "",
```

Comment:
This means seems `HELLOvimrc//` or `WORLDvimrc\\\\\` are also accepted. Is this trailing slashes/backslashes intentional? I wonder you ment `.*vimrc\\(/.*\\)\\?$`